crates/ruma-client-api/src/sync/sync_events/v4.rs

```diff
@@ -1,95 +1,95 @@
-//! [POST /_matrix/client/unstable/org.matrix.msc3575/sync](https://github.com/matrix-org/matrix-doc/blob/kegan/sync-v3/proposals/3575-sync.md)
+//! `POST /_matrix/client/unstable/org.matrix.msc3575/sync` ([MSC])
+//!
+//! Get all new events in a sliding window of rooms since the last sync or a given point of time.
```

Comment:
```suggestion
//! Get all new events in a sliding window of rooms since the last sync or a given point in time.
```
It sounds better to me, unless I'm mistaken.